src/app/dataset-view/additional-components/settings-component/settings.component.sass

```diff
@@ -0,0 +1,139 @@
+@import src/assets/styles/var
+p
+    margin: 0
+.custom-container
+    display: grid
+    grid-template-columns: 300px 1fr
+
+.content-container
+    padding: 15px 30px 0px
+    h2
+        font-weight: 400
+        margin: 0
+    dl
+        margin: 0
+
+.p-responsive
+    padding-left: 16px
+    a
+        text-decoration: none
+    .rename-btn
+        padding: 3px 12px
+        font-size: 14px
+        height: 32px
+        border: 1px solid rgba(27,31,36,0.25 )
+        border-radius: 6px
+        &:hover
+            background: rgba(27,31,36,0.16 )
+.fw-bold-custom
+    font-weight: 600
+.action-list
+    padding: 0
+
+.action-list-item
+    position: relative
+    list-style: none
+    border-radius: 6px
+    text-decoration: none
+
+.action-list-item a
+    &:hover
+        cursor: pointer
+        text-decoration: none
+
+.action-list-content
+    position: relative
+    display: grid
+    width: 100%
+    padding: 6px 8px
+    font-size: 14px
+    font-weight: 400
+    color: $app-color-fg-default
+    text-align: left
+    user-select: none
+    background-color: transparent
+    border: none
+    border-radius: 6px
+    transition: background 33.333ms linear
+    touch-action: manipulation
+    grid-template-rows: min-content
+    grid-template-areas: "leadingAction leadingVisual label trailingVisual trailingAction"
+    grid-template-columns: min-content max-content
+    align-items: start
+    &:hover
+        background: $app-color-action-list-item-default-selected-bg
+
+.action-list-item--nav-active
+    background: $app-color-action-list-item-default-selected-bg
+    &::after
+        position: absolute
+        top: calc(50% - 12px)
+        left: -8px
+        width: 4px
+        height: 24px
+        content: ""
+        background: $app-color-accent-fg
+        border-radius: 6px
+
+.action-list-item--navActive a
+    font-weight: 600
+
+.action-list-sectionDivider:empty
+    display: block
+    height: 1px
+    padding: 0
+    margin: 7px -8px 8px
+    list-style: none
+    background: $app-color-action-list-item-inline-divider
+    border: 0
+
+.action-list-sectionDivider:not(:empty)
+    display: flex
+    padding: 6px 8px
+    font-size: 12px
+    font-weight: 500
+    color: $app-color-fg-muted
+    flex-direction: column
```

Comment:
Same as above, I suggest to use sass nesting, and rename `sectionDivider` -> `section-divider`

```
.action-list-section-divider

  &:empty
  ...
  
  &:not(:empty)
  ...
```